packages/lodestar/test/sim/merge-interop.test.ts

```diff
@@ -273,16 +272,16 @@ describe("executionEngine / ExecutionEngineHttp", function () {
     });
   });
 
-  it("Pre-merge, run for a few blocks", async function () {
-    console.log("\n\nPre-merge, run for a few blocks\n\n");
-    const {genesisBlockHash} = await runGethPreMerge();
-    await runNodeWithGeth.bind(this)({
-      genesisBlockHash,
-      mergeEpoch: 1,
-      ttd: BigInt(terminalTotalDifficultyPreMerge),
-      testName: "pre-merge",
-    });
-  });
+  // it("Pre-merge, run for a few blocks", async function () {
```

Comment:
I am yet to run this runPreGeth test and confirm it working, I did add one comment on top of `runPreGeth` function. most probably it will work. 
This task was added in this tracker: https://github.com/ChainSafe/lodestar/issues/3425 to be picked once all feedback is resolved.

Reply:
resolved after coordinated a fix with the geth team over discord's interop channel :+1: 
test restored to original